models/stg_twilio.yml

```diff
@@ -473,10 +473,10 @@ models:
     tests:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
-            - category
-            - start_date
-            - end_date
-            - account_id
+            - category 
+            - start_date 
+            - end_date 
+            - account_id 
```

Comment:
Same not null test comment

Reply:
added not null tests to each of these fields